app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -39,6 +40,20 @@ export class DocumentSidePanel extends Component {
     }
   }
 
+  setDefaultDocumentToC(isEntity, documents, language, currentLanguage) {
```

Comment:
@mfacar this is a really minor thing but, can we change the name of this function to start with `get` instead of `set`?